extensions/ql-vscode/package.json

```diff
@@ -1834,7 +1826,6 @@
     "@types/unzipper": "^0.10.1",
     "@types/vscode": "^1.67.0",
     "@types/webpack": "^5.28.0",
-    "@types/webpack-env": "^1.18.0",
```

Comment:
I think this one should actually stay here: in the view code, we can use functions from the Webpack environment, such as `require.resolve`, `module.hot`, etc. I don't think we're currently using any of those, but we do have access to those.

Reply:
I'm happy to leave `@types/webpack-env` in there 👍🏼